tools/quantp/quantp.xml

```diff
@@ -0,0 +1,131 @@
+                    <label>Experiment design File (Please see the format below)</label>
+                </param>
+                <param name="method_type" type="select" label="Data summarization method" help="Perform T-Test on selecting Mean; Wilcoxon Ranksum Test on selecting Median">
+                    <option value="mean" selected="True">Mean (Default)</option>
+                    <option value="median">Median</option>
+                </param>
+                <param name="correction_method" type="select" label="Multiple testing correction method">
+                    <option value="BH" selected="True">Benjamini and Hochberg (BH) (Default)</option>
+                    <option value="holm">Holm</option>
+                    <option value="hochberg">Hochberg</option>
+                    <option value="hommel">Hommel</option>
+                    <option value="bonferroni">Bonferroni</option>
+                    <option value="BY">Benjamini and Yekutieli (BY)</option>
+                    <option value="none">None</option>
+                </param>
+                <param name="volcano_with" type="select" display="radio" label="Volcano plot with p-value or adjusted p-value">
+                    <option value="pval" selected="True">P-value (Default)</option>
+                    <option value="adj_pval">Adjusted P-value</option>
+                </param>
+            </when>
+        </conditional>
+        <param name="cooksd_upper" type="integer" value="4" optional="false" >
+            <label>Influential Observation cutoff: Observations > value * mean of Cook's distances (Default: "4" * mean(Cook's Distance))</label>
+        </param>
+        <param name="nclust" type="select" label="K-mean clustering: Number of clusters">
+            <option value="1">1</option>
+            <option value="2">2</option>
+            <option value="3">3</option>
+            <option value="4" selected="True">4</option>
+            <option value="5">5</option>
+            <option value="6">6</option>
+            <option value="7">7</option>
+            <option value="8">8</option>
+            <option value="9">9</option>
+            <option value="10">10</option>
+        </param>
+      
+        <param name="hm_nclust" type="select" label="Hierarchical clustering: Number of clusters (from Heatmap)">
+            <option value="1">1</option>
+            <option value="2">2</option>
+            <option value="3">3</option>
+            <option value="4">4</option>
+            <option value="5" selected="True">5</option>
+            <option value="6">6</option>
+            <option value="7">7</option>
+            <option value="8">8</option>
+            <option value="9">9</option>
+            <option value="10">10</option>
+        </param>
+    </inputs>
+    <outputs>
+        <data format="html" name="html_file" label="protein_transcript_correlation on ${pe_exp.name} and ${te_exp.name}.html"/>
```

Comment:
Why do you have underscores here: protein_transcript_correlation
And why is this .html at the end?